### PR TITLE
Added OS.get_system_time_msecs()

### LINKED ISSUE #19636: Getting system time in milli or micro seconds
**Godot version:**

3.1.dev

**OS/device including version:**

not related

**Issue description:**

we have `OS.get_system_time_secs()` for getting system time in seconds.
but no method for milli or micro seconds.
this is needed when making a game requires more precise time without game server but can be continue to play after quit and re-run.

**Steps to reproduce:**

**Minimal reproduction project:**

### PR DESCRIPTION
**Pull request by volzhs**
Fix #19636

### CODE REVIEW

**Comment by volzhs** [comment]
@akien-mga it's a pretty little feature can be added to 3.1 I think.

File: platform/windows/os_windows.cpp [hunk 1]
```diff
@@ -2142,8 +2142,13 @@ uint64_t OS_Windows::get_unix_time() const {
...
 
-	const uint64_t WINDOWS_TICK = 10000000;
-	const uint64_t SEC_TO_UNIX_EPOCH = 11644473600LL;
+	return get_system_time_msecs() / 1000;
+}
+
+uint64_t OS_Windows::get_system_time_msecs() const {
+
+	const uint64_t WINDOWS_TICK = 10000;
+	const uint64_t SEC_TO_UNIX_EPOCH = 11644473600000LL;
```
Comment (akien-mga) [inline platform/windows/os_windows.cpp]: That would be `MSEC_`.

Comment (volzhs) [reply platform/windows/os_windows.cpp]: updated

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**